### PR TITLE
Add type annotations to every function

### PR DESCRIPTION
**Pull request by melko**
This addresses issue #10 adding type annotations for every function in
the script.

### CODE REVIEW

File: vandor-rename.py [hunk 1]
```diff
@@ -152,23 +154,23 @@ def beg():
...
+    def assemble_classname() -> str:
         class_name = f'aula{class_n}exer{exer_n}'
 
         if evolution != '-1':
             class_name += 'Evolucao' + evolution
 
         return class_name
 
-    def get_new_name(typee, ext=None):
+    def get_new_name(typee: str, ext: str = None) -> str:
```
Comment (icaropires) [inline vandor-rename.py]: Hi ! Thanks for the PR :smiley: .
There's just one detail to change, `ext` param should also be [Optional](https://docs.python.org/3/library/typing.html#typing.Optional).

Could you fix it?

**Review by icaropires — CHANGES_REQUESTED** [review]

**Comment by icaropires** [comment]
@melko Could you please resolve the conflicts and add the requested changes?

**Comment by melko** [comment]
done

**Review by icaropires — APPROVED** [review]

**Review by icaropires — APPROVED** [review]
Great! Thanks for contributing :smiley:

**Pull request merged**